README.md

```diff
@@ -27,6 +27,18 @@ yarn add cypress-nextjs-auth0 --dev
 import 'cypress-nextjs-auth0';
 ```
 
+### Step 2: Register the task
+
+```js
+// cypress/plugins/index.js
+
+const encryptTask = require('cypress-nextjs-auth0/encrypt');
```

Comment:
I had to add a task because the libraries I use to do the JWE cookie encryption are Node only (I would update `jose` to v3 which supports browser too, but it needs Webpack 5 and Cypress is on 4)

Reply:
I agree. I think it's wise to rely on server-side code for the encrypt task if we have to pick between server or browser. I'm fine with the addition of the task but my comments on `encrypt.js` ask whether there is a better way to let developers include/overwrite the encrypt task. We can continue that conversation in those comments. 👍